frame/ibc/Cargo.toml

```diff
@@ -33,32 +33,36 @@ sp-std = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v
 sp-trie = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.22", default-features = false }
 
 hex-literal = { version = "0.3.4", optional = true }
-ibc = { git = "https://github.com/ComposableFi/ibc-rs", rev = "198440a62d4ccc88555cfef6253d6595319b88bf", default-features = false }
-ibc-proto = { git = "https://github.com/ComposableFi/ibc-rs", rev = "198440a62d4ccc88555cfef6253d6595319b88bf", default-features = false }
-ics23 = { version = "=0.8.0-alpha", default-features = false, optional = true }
-tendermint = { version = "0.23.7", optional = true, default-features = false }
-tendermint-proto = { version = "0.23.7", default-features = false }
+ibc = { git = "https://github.com/ComposableFi/ibc-rs", rev = "c7c73ab423445f501ceae3d1d5d74815b502744a", default-features = false }
+ibc-proto = { git = "https://github.com/ComposableFi/ibc-rs", rev = "c7c73ab423445f501ceae3d1d5d74815b502744a", default-features = false }
+ics23 = { git = "https://github.com/composablefi/ics23", branch = "seun-v0.8.0-alpha", default-features = false, optional = true }
+tendermint = { git = "https://github.com/composableFi/tendermint-rs", branch = "seun-0.23.7", default-features = false, optional = true }
+tendermint-proto = { git = "https://github.com/composableFi/tendermint-rs", branch = "seun-0.23.7", default-features = false }
 
 # local deps
 ibc-primitives = { path = "ibc-primitives", default-features = false }
 ibc-trait = { path = "ibc-trait", default-features = false }
 pallet-ibc-ping = { path = "../ibc-ping", default-features = false }
 
+#temporary dependencies
```

Comment:
Please describe why they are temporary, so other developers know when they can be removed.

Reply:
They'll be removed from this pallet in this PR https://github.com/ComposableFi/composable/pull/1122
because the runtime interface where they are used would be moved to it's own crate.

Reply:
Alright, but in the case that you randomly fall sick or whatever, let's add that.